tests/functional/detection_status/test_detection_status.py

```diff
@@ -90,6 +102,10 @@ def setUp(self):
   int* p = 0;
 
   i = *p + 42;
+
+  sizeof(42);
+  sizeof(42);
+  sizeof(42);
```

Comment:
Also create a test case to check the fix date. Resolve at least two bugs and check that the fixation date is the same.